### PR TITLE
Add more performance test scenarios

### PR DESCRIPTION
**Pull request by poorna2152**
## Purpose
Add the following performance test scenarios
- Data services (database read and write)
- Using File connector to write files using local file system
- HTTP request to JMS messaging
- HTTPS client -> MI/Bal -> HTTPS backend

### CODE REVIEW

File: distribution/scripts/ballerina/data_service/main.bal [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        sql:ExecutionResult[] result = check dbClient->batchExecute(insertQueries);
+        int|string? lastInsertId = result[0].lastInsertId;
+        if lastInsertId is int {
+            return lastInsertId;
+        }
+        return error("Unable to obtain last insert ID");
+    }
+
+    isolated resource function get [int id]() returns Order|error? {
+        log:printInfo(id.toString());
```
Comment (TharmiganK) [inline distribution/scripts/ballerina/data_service/main.bal:71]: Do we need this logging part?

File: distribution/scripts/ballerina/data_service/main.bal [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    isolated resource function get .() returns Order[]|error? {
+        Order[] orders = [];
+        stream<DBOrder, error?> resultStream = dbClient->query(
+            `SELECT * FROM Orders`
+        );
+        check from DBOrder dbOrder in resultStream
+            do {
+                orders.push(convertDBOrderToOrder(dbOrder));
+            };
+        check resultStream.close();
```
Comment (TharmiganK) [inline distribution/scripts/ballerina/data_service/main.bal:87]: @daneshk Do we need to close this stream from query result manually?

File: distribution/scripts/ballerina/data_service/main.bal [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    host = dbHost,
+    user = dbUser,
+    password = dbPassword,
+    port = dbPort,
+    database = dbName
+);
+
+service /'orders on securedEP {
+
+    isolated resource function post .(Request request) returns int|error? {
```
Comment (TharmiganK) [inline distribution/scripts/ballerina/data_service/main.bal:58]: Do we have to explicitly say this function as `isolated`? Check other places as well

File: distribution/scripts/ballerina/file_connector/main.bal [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        key: {
+            path: epKeyPath,
+            password: epKeyPassword
+        }
+    }
+);
+
+service /fileConnector on securedEP {
+    resource function post .(@http:Payload json payload) returns json|error {
+        check io:fileWriteJson(FILE_NAME, payload);
```
Comment (TharmiganK) [inline distribution/scripts/ballerina/file_connector/main.bal:21]: When we are running this in a docker environment, we need to have access to this directory to read, write and remove right? How do we configure the cloud options for these services?

File: distribution/scripts/ballerina/data_service/main.bal (hunk 2 above)
Comment (daneshk) [reply distribution/scripts/ballerina/data_service/main.bal:87]: As we are iterating through the whole stream and reading the complete resultset here, we don't need to close. We only need to close when we have not fully read the resultset.

File: distribution/scripts/ballerina/file_connector/main.bal (hunk 4 above)
Comment (poorna2152) [reply distribution/scripts/ballerina/file_connector/main.bal]: Assuming we are running in a vm and we don't have permission issues as per offline discussion we can keep it as it. Decided to make this a configurable

**Review by TharmiganK — APPROVED** [review]
HTTP services LGTM

**Review by SasinduDilshara — APPROVED** [review]

**Pull request merged**